src/test/test_switches.cpp

```diff
@@ -0,0 +1,315 @@
+/**
+ * This file is part of ExpressLRS
+ * See https://github.com/AlessandroAU/ExpressLRS
+ *
+ * Unit tests for over the air packet encoding, decoding and associated utilities
+ *
+ * Entry point is setup_switches()
+ */
+
+#include <Arduino.h>
+#include <unity.h>
+
+#include "LoRaRadioLib.h"
+#include "CRSF.h"       // has to come after LoraRadioLib.h for R9
+#include "targets.h"
+#include "common.h"
+#include <OTA.h>
+
+// Tests have only been run on TTGO V1, so we have to guard
```

Comment:
Hmmm, not ideal since these are on target tests, so can only be run by people that own the v1 TTGO... any tests are better than none I suppose, so leave it for now.

Reply:
Agreed. I wasn't sure how to fix it as I don't have any stm32 to run against. Running against a rx mini would also be painful as it doesn't have external reset, so the only thing that was really workable was an esp32. I used a spare wroom32 dev board for running the tests as my ttgo is in the radio.

Reply:
I can test on various targets once we merge